tests/runfiles/linux.run

```diff
@@ -181,8 +181,9 @@ tests = ['zfs_mount_001_pos', 'zfs_mount_002_pos', 'zfs_mount_003_pos',
     'zfs_mount_004_pos', 'zfs_mount_005_pos', 'zfs_mount_006_pos',
     'zfs_mount_007_pos', 'zfs_mount_008_pos', 'zfs_mount_009_neg',
     'zfs_mount_010_neg', 'zfs_mount_011_neg', 'zfs_mount_012_neg',
-    'zfs_mount_all_001_pos', 'zfs_mount_encrypted', 'zfs_mount_remount',
-    'zfs_multi_mount', 'zfs_mount_all_fail', 'zfs_mount_all_mountpoints']
+    'zfs_mount_all_001_pos', 'zfs_mount_all_fail', 'zfs_mount_all_mountpoints',
+    'zfs_mount_encrypted', 'zfs_mount_remount', 'zfs_mount_test_race',
+    'zfs_multi_mount']
```

Comment:
It looks like reordering these tests exposed some missing cleanup code which caused `zfs_mount_encrypted` and `zfs_mount_remount` to fail.

Reply:
Then I guess we just want to add `zfs_mount_test_race` at the end of this list ? This was reordered in alphabetical order per your suggestion.

Reply:
It would be nice to resolve whatever the cleanup issue is, since we should be able to reorder these tests.  But if that's not going to be straight forward, I'm fine with preserving the ordering for now.

Reply:
Updated